File: sub-model/src/main/java/org/github/base/entity/QrtzCalendarsEntity.java
package org.github.base.entity;

import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.Setter;
import org.github.base.AbstractEntity;

import java.sql.Blob;

/**
 * <p>
 * 
 * </p>
 *
 * @author JYD_XL
 * @since 2019-11-21
 */
@Setter
@Getter
@EqualsAndHashCode(callSuper = true)
@TableName("qrtz_calendars")
public class QrtzCalendarsEntity extends AbstractEntity {
    /** UID */
    private static final long serialVersionUID = 1L;

    @TableId("sched_name")
    private String schedName;

    @TableField("calendar_name")
    private String calendarName;

    @TableField("calendar")
    private Blob calendar;
}

File: sub-model/src/main/java/org/github/base/entity/QrtzCalendarsEntity.kt
package org.github.base.entity

import com.baomidou.mybatisplus.annotation.TableName
import org.github.base.AbstractEntity
import java.sql.Blob

/**
 * <p>
 * 
 * </p>
 *
 * @author JYD_XL
 * @since 2019-11-19
 */
@TableName("qrtz_calendars")
class QrtzCalendarsEntity: AbstractEntity() {

    var schedName: String? = null

    var calendarName: String? = null

    var calendar: Blob? = null

}
